e2e/qgb/deployer_test.go

```diff
@@ -38,9 +38,9 @@ func TestDeployer(t *testing.T) {
 	HandleNetworkError(t, network, err, false)
 
 	// FIXME should we use the evm client here or go for raw queries?
-	evmClient := orchestrator.NewEvmClient(nil, bridge, nil, network.EVMRPC)
+	evmClient := orchestrator.NewEvmClient(nil, bridge, nil, network.EVMRPC, orchestrator.DEFAULTEVMGASLIMIT)
```

Comment:
should we be using the configured gas limit here, not the default?

Reply:
this is for e2e, doesn't matter much

Reply:
this is for e2e, doesn't matter much

Reply:
this is for e2e, doesn't matter much

Reply:
lol whoops! okie dokie